### PR TITLE
Updated sv.po to match en.po. See issues #271.

### PR DESCRIPTION
**Pull request by matsduf**

### CODE REVIEW

**Comment by sandoche2k** [comment]
@matsduf seems Ok for me, even thought i cannot vouch for the Swedish translation.

File: share/sv.po [hunk 1]
```diff
@@ -111,6 +111,7 @@ msgstr "Namnservern {ns}/{address} svarade inte på frågan om 'NS'."
 
 #: CONNECTIVITY:IPV4_DISABLED
 #: CONNECTIVITY:IPV6_DISABLED
```
Comment (mattias-p) [inline share/sv.po]: I suggest grouping these with BASIC:IPV4_DISABLED and BASIC:IPV6_DISABLED respectively instead. If we organize the po file that way standard gettext tools won't break our conventions.

File: share/sv.po [hunk 2]
```diff
@@ -291,6 +292,7 @@ msgstr "Delegeringen i moderzonen har en eller flera namnservrar som inte listas
 
 #: DELEGATION:IPV4_DISABLED
 #: DELEGATION:IPV6_DISABLED
```
Comment (mattias-p) [inline share/sv.po]: I suggest grouping these with BASIC:IPV4_DISABLED and BASIC:IPV6_DISABLED respectively.

File: share/sv.po (hunk 1 above)
Comment (mattias-p) [inline share/sv.po]: I suggest that these two be grouped with BASIC:IPV4_DISABLED and BASIC:IPV6_DISABLED respectively. That way standard po tooling could be used without breaking our our conventions for organizing po files (by removing this newline again). Also, just having them lying around like this makes it look like they were forgotten.

File: share/sv.po (hunk 2 above)
Comment (mattias-p) [inline share/sv.po]: I suggest that these two be grouped with BASIC:IPV4_DISABLED and BASIC:IPV6_DISABLED respectively.

File: share/sv.po [hunk 3]
```diff
@@ -429,7 +431,7 @@ msgstr "Det finns ingen DS-post med matchande DNSKEY-post."
 
 #: DNSSEC:DS_RFC4509_NOT_VALID
 msgid  "Existing DS with digest type 2, while they do not match DNSKEY records, prevent use of DS with digest type 1 (RFC4509, section 3)."
-msgstr "Existing DS with digest type 2, while they do not match DNSKEY records, prevent use of DS with digest type 1 (RFC4509, section 3)."
+msgstr "Befintlig DS med 'digest' typ 2 som inte stämmer med DNSKEY-posten, förhindrar ändå, i enlighet med RFC 4509 (stycke 3), att DS med 'digest' typ 1."
```
Comment (mattias-p) [inline share/sv.po]: I think this one needs a little bit more love.

File: share/sv.po [hunk 4]
```diff
@@ -637,6 +639,7 @@ msgstr "Följande namnservrar stöder EDNS0: {names}."
 
 #: NAMESERVER:IPV4_DISABLED
 #: NAMESERVER:IPV6_DISABLED
```
Comment (mattias-p) [inline share/sv.po]: I suggest that these two be grouped with BASIC:IPV4_DISABLED and BASIC:IPV6_DISABLED respectively.

File: share/sv.po [hunk 5]
```diff
@@ -778,12 +781,7 @@ msgid  "Both ends of all labels of the domain name ({name}) have no hyphens."
 msgstr "Domännamnet '{name}' har ingen domändel som börjar eller slutar med ett bindestreck."
 
 #: SYNTAX:NO_RESPONSE_MX_QUERY
-msgid  "No response from nameserver(s) on MX queries."
-msgstr "Inget svar gavs på fråga om MX-record."
-
 #: SYNTAX:NO_RESPONSE_SOA_QUERY
```
Comment (mattias-p) [inline share/sv.po]: I suggest that these two be grouped with ZONE:NO_RESPONSE_MX_QUERY and ZONE:NO_RESPONSE_SOA_QUERY respectively.

File: share/sv.po [hunk 6]
```diff
@@ -958,7 +956,13 @@ msgid  "No target (MX, A or AAAA record) to deliver e-mail for the domain name."
 msgstr "Inget mål att levera mail till (MX-, A- eller AAAA-post) är angivet för domänen."
 
 #: ZONE:NO_RESPONSE_MX_QUERY
+msgid  "No response from nameserver(s) on MX queries."
+msgstr "Inget svar gavs på fråga om MX-record."
```
Comment (mattias-p) [inline share/sv.po]: MX-post?

File: share/sv.po [hunk 7]
```diff
@@ -958,7 +956,13 @@ msgid  "No target (MX, A or AAAA record) to deliver e-mail for the domain name."
...
+
 #: ZONE:NO_RESPONSE_SOA_QUERY
+msgid  "No response from nameserver(s) on SOA queries."
+msgstr "Inget svar gavs på fråga om SOA-record."
```
Comment (mattias-p) [inline share/sv.po]: SOA-post?

**Comment by matsduf** [comment]
@mattias-p, fixed all proposed changes. Grouped under first instance.

File: share/sv.po [hunk 8]
```diff
@@ -431,7 +423,7 @@ msgstr "Det finns ingen DS-post med matchande DNSKEY-post."
 
 #: DNSSEC:DS_RFC4509_NOT_VALID
 msgid  "Existing DS with digest type 2, while they do not match DNSKEY records, prevent use of DS with digest type 1 (RFC4509, section 3)."
-msgstr "Befintlig DS med 'digest' typ 2 som inte stämmer med DNSKEY-posten, förhindrar ändå, i enlighet med RFC 4509 (stycke 3), att DS med 'digest' typ 1."
+msgstr "Befintlig DS med 'digest' typ 2, som inte stämmer med DNSKEY-posten, förhindrar ändå, i enlighet med RFC 4509 (stycke 3), att DS med 'digest' typ 1 används."
```
Comment (mattias-p) [inline share/sv.po]: Det är fortfarande väldigt stakigt med många kommatecken och insprängda bisatser. Här är ett förslag på en ren uträtning: "Befintlig DS med 'digest' typ 2 förhindrar att DS med 'digest' typ 1 används. Detta trots att den förra inte stämmer med DNSKEY-posten. (RFC 4509 stycke 3)"

Om jag å andra sidan bara kollar i RFC:n tycker jag det ser ut som att man kan lämna värderingen till severity-nivån och bara skriva meddelandet som: "DS med 'digest' typ 1 bör ej användas tillsammans med DS med 'digest' typ 2. (RFC 4509 stycke 3)"

Men jag har kanske missat något viktigt.

File: share/sv.po [hunk 9]
```diff
@@ -109,9 +109,6 @@ msgstr "Namnservern {ns}/{address} svarade inte med några NS-poster. RCODE var
 msgid  "Nameserver {ns}/{address} did not respond to NS query."
 msgstr "Namnservern {ns}/{address} svarade inte på frågan om 'NS'."
```
Comment (mattias-p) [inline share/sv.po]: The quoting of ns/address pairs is inconsistent. If that's intentional or if fixing that is out of scope for this issue, I'm fine with that.

Comment (matsduf) [reply share/sv.po]: Firstly, I think it is out of scope. Secondly, we need rules for quoting to be consistent. We need to create those.

Comment (pawal) [reply share/sv.po]: Fix the consistency of raw log output (i.e. in JSON) before fixing consistency in language and formatting of addresses and such. It will make it easier. https://github.com/dotse/zonemaster-engine/issues/60
